### PR TITLE
[SA-CORE-2019-007] Update drupal/core to 7.67 (from 7.66)

### PR DESCRIPTION
**Pull request by tobybellwood**
## https://www.drupal.org/sa-core-2019-007

Project:  Drupal core
Date:  2019-May-08
Security risk:  Moderately critical 14∕25 AC:Complex/A:Admin/CI:All/II:All/E:Theoretical/TD:Uncommon
Vulnerability:  Third-party libraries
Description:  This security release fixes third-party dependencies included in or required by Drupal core. As described in TYPO3-PSA-2019-007: By-passing protection of Phar Stream Wrapper Interceptor:

In order to intercept file invocations like file_exists or stat on compromised Phar archives the base name has to be determined and checked before allowing to be handled by PHP Phar stream handling. [...]

The current implementation is vulnerable to path traversal leading to scenarios where the Phar archive to be assessed is not the actual (compromised) file.

Solution: 
Install the latest version:

If you are using Drupal 7, update to Drupal 7.67.

## https://www.drupal.org/project/drupal/releases/7.67

Release notes
Maintenance and security release of the Drupal 7 series.

This release fixes security vulnerabilities. Sites are urged to upgrade immediately after reading the notes below and the security announcement:

Drupal core - Third Party Libraries - SA-CORE-2019-007
No other fixes are included.

Important update information
No changes have been made to the .htaccess, web.config, robots.txt or default settings.php files in this release, so upgrading custom versions of those files is not necessary if your site is already on the previous release.

### CODE REVIEW

File: drupal-org.make [hunk 1]
```diff
@@ -133,7 +133,7 @@ projects[search_api_page][patch][] = "https://www.drupal.org/files/issues/290079
 projects[search_api_solr][version] = "1.14"
 projects[seckit][version] = "1.9"
 projects[service_links][version] = "2.3"
-projects[services][version] = "3.23"
+projects[services][version] = "3.24"
```
Comment (pandaskii) [inline drupal-org.make]: Already having a PR https://github.com/govCMS/govCMS/pull/868

**Review by pandaskii — CHANGES_REQUESTED** [review]
Looks good. Buggy CI is not working randomly

File: drupal-org.make (hunk 1 above)
Comment (tobybellwood) [reply drupal-org.make]: This PR *has* to go against the release branch for use in govcmsagoon - we don't have the flexibility to reference a commit here. https://github.com/govCMS/govcmslagoon/blob/release/govcmslagoon-3.6.0/.env.default#L38

**Review by pandaskii — APPROVED** [review]
Looks good, thanks

**Review by pandaskii — APPROVED** [review]
Looks good, thanks

**Pull request merged**